AUTHORS.md

```diff
@@ -5,13 +5,13 @@ during [an NSF funded postdoc](http://www.leouieda.com/blog/hawaii-gmt-postdoc.h
 with [Paul Wessel](http://www.soest.hawaii.edu/wessel) at the University of Hawaii at
 Manoa.
 
-The following people have contributed code to the project (alphabetical by last name)
+The following people have contributed code to the project (alphabetical by name)
 and are considered the "PyGMT Developers":
```

Comment:
```suggestion
and are considered to be "PyGMT Developers":
```
Small change, but I think that this makes it read less like an exhaustive list of PyGMT developers. The list shouldn't be considered exhaustive because one of the main ways that people indicate that they do not want to be included in the Zenodo archives is by removing themselves from ```AUTHORS.md``` (as detailed in ```AUTHORSHIP.md```).

Reply:
Cool, great attention to detail! I'm made the change at 67b7840668f88173e68ab383ed85d143d929ad1a. Feel free to add your name and ORCID to the list too @meghanrjones, you've helped a lot with setting up the Slack channel, PyGMT sub-forum and many more. Plus I'm sure we'll see more contributions from you soon!